tests/minimax.wcon

```diff
@@ -27,7 +27,10 @@
 		"ox":"mm",
 		"speed":"mm/s",
 		"curvature":"1/mm",
-		"width":"mm"
+		"width":"mm",
+		"humidity":"%",
+		"temperature":"C",
+        "age":"h"
```

Comment:
Indentation doesn't seem to match here.  Formatting everything with spaces helps keep things looking consistent for everyone.